docs/reference/pip_download.rst

```diff
@@ -42,10 +55,66 @@ Options
 Examples
 ********
 
-#. Download a package and all of its dependencies
-
-    ::
-
-      $ pip download SomePackage
-      $ pip download -d . SomePackage  # equivalent to above
-      $ pip download --no-index --find-links=/tmp/wheelhouse -d /tmp/otherwheelhouse SomePackage
+1. Download a package and all of its dependencies
```

Comment:
It's better to use `#.` notation to let sphinx do the numbering (so you don't have to modify all the following examples when you delete the first one)